docs/versioned_docs/version-3.2/guides/multitrack-progress.md

```diff
@@ -94,12 +94,12 @@ import { useProgressStateStore } from '../store';
 const setProgress = useProgressStateStore.getState().setProgress;
 
 export const PlaybackService = async function() {
-  TrackPlayer.addEventListener(Event.PlaybackProgressUpdated, async () => {
+  TrackPlayer.addEventListener(Event.PlaybackProgressUpdated, async (event) => {
     // get the position and currently playing track.
-    const position = TrackPlayer.getPosition();
-    const track = TrackPlayer.getCurrentTrack();
+    const track = await TrackPlayer.getActiveTrack();
```

Comment:
`getProgress` isn't available in `v3.2` (which this doc is for) so we don't want to update this one. It should stay as is I believe. same with `getActiveTrack`.